dev-packages/e2e-tests/test-applications/aws-lambda-layer-cjs/src/run-lambda.js

```diff
@@ -0,0 +1,7 @@
+const { handle } = require('./lambda-function');
+const event = {};
+const context = {
+  invokedFunctionArn: 'arn:aws:lambda:us-east-1:123453789012:function:my-lambda',
+  functionName: 'my-lambda',
+};
```

Comment:
minimal event and context objects that are usually passed to the lambda function by AWS